### PR TITLE
Effects of (G-)Max Moves should not be secondaries

### PR DESCRIPTION
**Pull request by urkerab**

### CODE REVIEW

**Comment by KrisXV** [comment]
There should be a `secondary: null,` property on all of these moves if this is the case

**Comment by Zarel** [comment]
Huh, I'm surprised `secondary: null` is required.

File: data/moves.js [hunk 1]
```diff
@@ -7146,14 +7143,7 @@ let BattleMovedex = {
...
 		isMax: "Orbeetle",
-		secondary: {
-			chance: 100,
-			self: {
-				onHit() {
-					this.field.addPseudoWeather('gravity');
-				},
-			},
-		},
+		pseudoweather: 'gravity',
```
Comment (Zarel) [inline data/moves.js]:
```suggestion
		pseudoWeather: 'gravity',
```

Comment (urkerab) [reply data/moves.js]: Huh, and I thought I'd tested all of the moves. I must be blind...

**Comment by urkerab** [comment]
`secondary: null` (or its predecessor `secondary: false`) doesn't seem to have been necessary for years. Commit 21be1f4 centralised secondary handling, but even back then, I can't see anything checking for a specific value.

**Pull request merged**